### ISSUE TITLE
Refreshing issue while returning from another tabs

### ISSUE THREAD

**Issue by shilpabuddha4** [JohnMcLear/draw#237]
Hi John,
Good Work!!
This is a nice board with good options.

But, I have some loading issue with drawing board, when i open other tabs in the browser and returning to the board. The board is completely becoming empty when i return from another tabs to board. Can you please let me know how to fix this issue. If i try to reload the URL then only the complete board is displaying as it is before.

**Comment by JohnMcLear** [comment]
I assume some disconnect logic needs adding.  See if a socket disconnect event is fired and/or reconnect and add some logic to reload the board on reconnect.

**Comment by bytesnz** [comment]
Due to Chrome tab discarding. As a workaround, you can disable it in Chrome https://productforums.google.com/forum/m/#!topic/chrome/9mmeqjl5BvA

**Comment by bytesnz** [comment]
Hey @shilpabuddha4, what browser are you using? If using chrome, are you still having the issue (if you have upgraded to the latest version)

**Comment by theoryshaw** [comment]
I've had the same issue, for some time, as well.  I just reload--not ideal, however.

**Comment by bytesnz** [comment]
Yeah, it's been an issue for quite a long time. Hopefully, I'll get a little bit of time to look at this.
